department-of-veterans-affairs/caseflow
# Record worksheet views

Connects #3816

Similar functionality in Reader: https://github.com/department-of-veterans-affairs/caseflow/commit/85d489f0f4de0c1c0769cdebfd3efd1bee52cbf7.

## Test
1. Go to the daily docket page.
1. View a worksheet.
1. Refresh the daily docket page.
1. See that the name of the vet is unbolded.

![image](https://user-images.githubusercontent.com/25331532/34175857-cbe0968c-e4cb-11e7-9d5a-7420829dab21.png)

## TODO
Add tests. I'll do that once @sharonwarner signs off on the approach.
